tests/test_multiline.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+from mssqlcli.mssqlbuffer import _is_complete
+
+
+class TestMssqlCliMultiline:
+    testdata = [
+        (None, False),
+        ('', False),
+        ('select 1 /* open comment!\ngo', False),
```

Comment:
how about some tests where you have a go at the beginning followed by some select statement